lighty-modules/lighty-infrautils/pom.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>lighty-modules-aggregator</artifactId>
```

Comment:
Never derive from aggregator -- this should be pointing to lighty-parent, in which case ...

Reply:
this module is deleted in later commits